site/content/docs/latest/reference/developer/release-process.md

```diff
@@ -145,38 +154,42 @@ cargo update
 
-To update the website engine, you need to Update the `HUGO_VERSION` variable in the [netlify.toml](https://github.com/vmware-tanzu/kubeapps/blob/main/site/netlify.toml) to the latest [Hugo release](https://github.com/gohugoio/hugo/releases/).
+To update the website engine, you need to Update the `HUGO_VERSION` variable in the [netlify.toml](https://github.com/vmware-tanzu/kubeapps/blob/main/site/netlify.toml) 
+to the latest [Hugo release](https://github.com/gohugoio/hugo/releases/).
 
 ## 1 - Select the commit to be tagged and perform some tests
 
-Once the dependencies have been updated and the chart changes merged, the next step is to choose the proper commit so that we can base the release on it. It is, usually, the latest commit in the main branch. Then, some manual and automated tests should be performed to ensure the stability and reliability of the release.
+Once the dependencies have been updated and the chart changes merged, the next step is to choose the proper commit so that we can base the release on it. 
+It is, usually, the latest commit in the main branch. Then, some manual and automated tests should be performed to ensure the stability and reliability of the release.
 
-## 1.1 - Trigger a `prerelease` CI flow
+## 1.1 - Trigger the `Full Integration Pipeline` workflow (former `prerelease` flow in `CircleCI`)
 
-One of the CI flows we have defined is the `prerelease` one. It is being triggered once a commit is pushed to the `prerelease` branch in the Kubeapps repository. Although the precise instructions may differ depending on your git configuration, the main steps are:
+The `Full Integration Pipeline` workflow in GitHub Actions is a manual workflow that runs the full pipeline, including those end-to-end tests
+that are run on external k8s clusters, usually managed by a cloud provider (currently only GKE). To trigger this workflow you have to follow
+the following steps:
 
-```bash
-# assuming you are in the main branch, with the latest changes pulled locally
-# and you already have a `prerelease` local branch
-git checkout prerelease
-git merge main
-git push origin prerelease # replace `origin` by your remote name
-```
-
-Then, check out the workflow that has just been created in CircleCI: [https://app.circleci.com/pipelines/github/vmware-tanzu/kubeapps?branch=prerelease](https://app.circleci.com/pipelines/github/vmware-tanzu/kubeapps?branch=prerelease).
+1. Go to the [Actions section](https://github.com/vmware-tanzu/kubeapps/actions) in the Kubeapps GitHub repository.
+2. Click on `Full Integration Pipeline` in the list of workflows on the left side of the screen.
+3. Click on the `Run Workflow` dropdown selector at the right-top of the list of workflow runs.
+4. Select `main` (the default one) in the list of branches.
+5. Click the `Run workflow` button.
+6. Check out the workflow run that has just been created.
```

Comment:
I'm not sure if you mean "check out" as in a git check out, or just "take a look". If the latter, perhaps:

```suggestion
6. You can then interact with and watch the workflow run that has just been created.
```